### PR TITLE
fix(api): error when token passed is empty

### LINKED ISSUE #79: bug(api): Ksctl Cred accepting empty strings
# Describe 🐞 

``Login Success`` even if no values are passed 

# Reproduce 💻 ➡️  💻 

``ksctl cred``

# Possible Fix & Expected behavior🔧 

# Screenshots :framed_picture:

<pre>Enter your credentials
Enter your SUBSCRIPTION ID 👇
    Enter Secret-&gt; 
Enter your TENANT ID 👇
    Enter Secret-&gt; 
Enter your CLIENT ID 👇
    Enter Secret-&gt; 
Enter your CLIENT SECRET 👇
    Enter Secret-&gt; 
2023/03/20 07:48:56 💾 configuration
Login Success
</pre>

**Desktop (please complete the following information):**
 - OS
   - [x] MacOS
   - [x] Windows
   - [x] Linux
 - ksctl version `<Version>`

# Additional context

### PR DESCRIPTION
**Pull request by riteshsonawane1372**
# Tasks / Issue :construction: 🔧 

- [x] #79 

# Solution :heavy_check_mark:

# Screenshots :framed_picture:

# Note to reviewers :notebook:

### CODE REVIEW

**Comment by riteshsonawane1372** [comment]
@dipankardas011 Have tested it working good

File: api/utils/main.go [hunk 1]
```diff
@@ -478,6 +478,10 @@ func UserInputCredentials() (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if len(bytePassword)==0{
+		fmt.Println("Secret is empty!")
```
Comment (dipankardas011) [inline api/utils/main.go:482]: You can use here logger.err()
Also when the password is empty the program should terminate and no file write had happened

**Review by dipankardas011 — CHANGES_REQUESTED** [review]

File: api/utils/main.go (hunk 1 above)
Comment (riteshsonawane1372) [reply api/utils/main.go:482]: Shouldn't we give the user another chance to enter the secrets?

Comment (dipankardas011) [reply api/utils/main.go:482]: Not sure if it affects usability 🤔

Comment (dipankardas011) [reply api/utils/main.go:482]: Okay let's get user another chance

File: api/utils/main.go [hunk 2]
```diff
@@ -471,13 +471,17 @@ func (sshPayload *SSHPayload) SSHExecute(logging logger.Logger, flag int, script
...
+func UserInputCredentials(logging logger.Logger) (string, error) {
 
 	fmt.Print("    Enter Secret-> ")
 	bytePassword, err := terminal.ReadPassword(int(os.Stdin.Fd()))
 	if err != nil {
 		return "", err
 	}
+	if len(bytePassword) == 0 {
+		logging.Err("Empty secret passed!")
+		UserInputCredentials(logging)
```
Comment (dipankardas011) [inline api/utils/main.go:483]: there should be return statement?

**Review by dipankardas011 — CHANGES_REQUESTED** [review]

**Review by dipankardas011 — APPROVED** [review]
LGTM

**Comment by dipankardas011** [comment]
@riteshsonawane1372 the your commits must be signed

**Pull request merged**